test/service/test_request_validation.py

```diff
@@ -328,3 +298,33 @@ def test_size(self):
             with self.subTest(size=size):
                 url.args.set('size', size)
                 test(url, arg)
+
+
+@patch_dss_source
+@patch_source_cache
+class RequestParameterResponseTest(WebServiceTestCase):
```

Comment:
Why isn't this in `RequestParameterValidationTest`?

Reply:
`RequestParameterValidationTest` is composed of tests that don't require the indices set up. Instead of changing the existing class I thought a new class would be cleaner.